pkg/subnet/byte_set.go

```diff
@@ -0,0 +1,101 @@
+package subnet
+
+import "fmt"
+
+// ByteSet represents 0 - 255 unique bytes.
+type ByteSet [4]uint64
```

Comment:
This comment and that type did not give me much of any idea what this is/does.

Perhaps:

```go
// Mask256 is a 256-bit bitmask.
type Mask256 [4]int64
```